deploy/release.groovy

```diff
@@ -48,6 +48,26 @@ def buildImage(imageName){
     }
 }
 
+def getStandaloneImage(imageName){
+    stage('build standalone npm') {
+        dir('runtime'){
+            container('ui'){
+               sh 'npm cache clean --force'
```

Comment:
Not required in a fresh environment, unless you have found a reason why it's needed.

Reply:
@debloper I don't think the environment is fresh when we rebuild the PR. At times PR build fails due to npm `integrity` errors. This command makes sure the environment is fresh before we start building the PR.